src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
```

Comment:
[CODE KNOWLEDGE] - Instead of using the `function` keyword, use the method shorthand syntax for defining methods within the object.